config.yaml

```
@@ -73,4 +73,29 @@ Standards:
         Type: Checkpoint
         UID: cebb89a0-61f0-4bc0-a889-6654045c839c
         Path: /05-pse/01-pse-05.checkpoint.md
+  -
+    Title: PSE 06
+    Description: Merging Sorted Lists
+    UID: 3253d4dd-bf5c-484b-8481-2d85f3844213
+    SuccessCriteria:
+      - success criteria
+    ContentFiles:
+      -
+        Type: Checkpoint
+        UID: cebb89a0-61f0-4bc0-a889-6654045c839c
```

This is a duplicate ID with the content/checkpoint file on PSE 05 (line 74)

```suggestion
        UID: 4df15cec-4b08-4c80-a10b-5ab6488b1a44
```

This one will work!